src/main/java/com/ericsson/ei/controller/LoginControllerImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.ericsson.ei.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.json.JSONObject;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+@Component
+@CrossOrigin
+@Api(value = "Auth", description = "REST endpoints for authentication and authorization")
+public class LoginControllerImpl implements LoginController {
+
+    @Override
+    @CrossOrigin
+    @ApiOperation(value = "To login user", response = String.class)
+    public ResponseEntity<?> getAuthLogin() {
+        try {
+            String currentUser = SecurityContextHolder.getContext().getAuthentication().getName();
+            JSONObject json = new JSONObject();
+            json.put("user", currentUser);
+            return new ResponseEntity<>(json.toString(), HttpStatus.OK);
```

Comment:
instead of initializing json object we can manage with string Like **"{'user':'"+currentUser+"'}"**